## Abstract

Large scale genetic screening is tedious and time-consuming for phenotypes that require complicated analysis. In an effort to address this problem, we propose a novel two-tiered screening system using a collection of UAS lines covering most *Drosophila* miRNAs <strike>that are all inserted into a common genomic landing site via site-specific recombination</strike>. Under this system, an initial “pooling” screen identifies a miRNA or miRNAs whose tissue-specific overexpression causes a phenotype of interest. Since miRNAs inhibit translation or direct the destruction of their target mRNAs, any phenotype observed with miRNA overexpression should be attributable to the loss-of-function of one or more target mRNAs.

MicroRNA-target pairing is determined via complementary base pairing through a short seed sequence. This means bioinformatic target prediction using the seed sequence of each positive hit in our primary screen provides a list of candidates for a secondary RNAi-based screen. These predicted miRNA targets can be prioritized based on expression pattern, and if multiple miRNAs give the same phenotype, overlapping target predictions can be given higher priority in the follow-up screen. This two-tiered system allows us to rapidly close in on individual target genes involved in a phenomenon of interest in sometimes < 200 crosses instead of the thousands required for a typical full-coverage RNAi screen.

We have demonstrated the effectiveness of this screening technique by identifying miRNAs that alter odorant receptor expression. Subsequent miRNA target prediction and follow-up screening has validated a role for the transcription factor Atf3 in insect odorant receptor expression.