## Introduction

The development of libraries of transgenic RNAi fly stocks (e.g., the [Vienna *Drosophila* RNAi library](http://stockcenter.vdrc.at/control/rnailibrary) and the [Transgenic RNAi Project (TRiP)](http://www.flyrnai.org/TRiP-HOME.html)) has been a tremendous boon to the *Drosophila* community because they permit tissue-specific knockdown of almost all genes in the genome. These resources make it possible to perform genome-wide screens for genes associated with almost any phenotype of interest. Unfortunately, the sheer size of these libraries--often upwards of 20,000 stocks--means that performing such screens remains labor-intensive and tedious. In this paper, we describe our development of a two-tiered screening system comprising an initial pooling screen using miRNA over-expression that generates a list of candidate genes involved in a phenotype of interest and a secondary screen using gene-specific RNAi that narrows this list of candidates to a single responsible target gene.

MicroRNAs are short, endogenous, single-stranded RNA molecules that act in the context of a large protein complex to either inhibit translation or induce the degradation of target mRNAs. Since the miRNA-target mRNA relationship is determined largely by a short ~9 nucleotide seed sequence that may occur in many copies scattered over the genome, many miRNAs are capable of down-regulating multiple targets. The specificity of the relationship between a miRNA seed sequence and its complements in the open reading frames and 3'-untranslated regions (3'-UTRs) of target mRNAs has led several research groups to develop bioinformatic algorithms that convert mature miRNA sequences into lists of potential mRNA targets. These lists of candidate targets, however, are plagued by large numbers of false positives partly because the algorithms that generate them cannot fully account for the precise spatial and temporal patterns of miRNA and target mRNA expression. In other words, a miRNA may be capable of down-regulating a particular target and never actually do so because the two are never simultaneously expressed in the same tissue. It follows conversely that miRNA overexpression in arbitrary tissues using the binary GAL4/UAS expression system would likely lead to artifactual miRNA-target mRNA pairings. Rather than seeing such artifacts as a potential drawback of using a library of UAS-miRNA stocks, we propose that such pairings can be useful as part of a two-tiered screening system.



Since we initiated this project to create a library of transgenic UAS-miRNA fly stocks that permit tissue-specific miRNA overexpression, two other groups have published similar resources \cite{aleigh_Kim_Ni_Duan_et_al__2012, shauser_Forstemann_Basler_2012}. 

Recently Jafari et al. reported the results of a large-scale RNAi screen that identified seven transcription factors, various permutations of which determine the odorant receptor expressed by each population of olfactory neurons in adult *Drosophila*. Despite the success of their screen, Jafari et al. extrapolated from the complexity of the fly olfactory system and estimated that at least three more unidentified transcription factors are likely part of the combinatorial code that determines OR expression \cite{Brochtrup_Hummel_Alenius_2012}. We thus decided that the fly olfactory system would be a good proving ground for our two-tiered miRNA-based screening concept. Identifying miRNAs whose overexpression alters odorant receptor expression would allows attempt to identify these remaining transcription factors that drive odorant receptor expression.