## Introduction

The development of libraries of transgenic RNAi fly stocks (e.g., the Vienna RNAi library and the Bloomington TRiP library) has been a tremendous boon to the Drosophila community because they permit tissue-specific knockdown of almost all genes in the genome. These resources make it possible to perform genome-wide screens for genes associated with almost any phenotype of interest. Unfortunately, the sheer size of these libraries--often upwards of 20,000 stocks--means that performing such screens remains labor-intensive and tedious. In this paper, we describe our development of a two-tiered screening system comprising an initial pooling screen using miRNA over-expression that generates a list of candidate genes involved in a phenotype of interest and a secondary screen using gene-specific RNAi that narrows this list of candidates to a single responsible target gene.

MicroRNAs are short (i.e., ~22 nucleotide), endogenous, single-stranded RNA molecules that act in the context of a large protein complex to either inhibit translation or induce the degradation of target mRNAs. Since the miRNA-target mRNA relationship is determined largely by a short ~9 nucleotide seed sequence that may occur in many copies scattered over the genome, many miRNAs are capable of down-regulating multiple targets. The specificity of the relationship between a miRNA seed sequence and its complements in the open reading frames and 3'-untranslated regions (3'-UTRs) of target mRNAs is sequence-specific, several groups have developed bioinformatic algorithms that convert miRNA sequences into lists of potential targets. These lists of candidate targets, however, are plagued by large numbers of false positives partly because the algorithms that generate them cannot fully account for the precise spatial and temporal patterns of miRNA and target mRNA expression.

In other words, a miRNA may be capable of down-regulating a particular target and never get the chance because the two are never simultaneously expressed in the same tissue. It follows conversely that miRNA overexpression in arbitrary tissues using the popular GAL4/UAS system would likely lead to artifactual miRNA-target mRNA pairings. Rather than seeing such artifacts as a potential drawback of using a library of UAS-miRNA stocks, we propose that such pairings can be quite useful.

---

Most organisms live in complex environments in which food, danger and potential mates must be detected by chemical clues. Thus, the olfactory system of these organisms are necessarily complex themselves. The olfactory system of the vinegar fly Drosophila melanogaster the stands out as example for the complexity of odor recognition system. Drosophila melanogaster, with its well established tools and protocols, provides an excellent model system to unravel the mysteries of olfaction. Not only is the hierarchial organization of the fly olfatory system similar to that of mammals, it is also simpler having 62 Or genes, Fly antennae detect air borne odorants in sensory sensilla that contain one to four olfactory sensory neurons. Sensory neurons expressing a specific odorant receptors are found in the same part of the antenna and project to the same glomerulus and glomeruli in the antennal lobe. The precise connectivity of the odor map with a large number of olfactory sensory neurons and tight regulation of their expression in a patchy manner throughout the fly antenna raises significant questions on the molecular mechanisms of its regulation.

The two transcription factors- Acj6 and pdm3, have been shown to specify a subset of Drosophila ORs. Large scale RNAi screen identified the seven transcription factors that are necessary for the OR  expression.

To lower the effort and time spent on large scale RNAi screen, we propose a novel two-tiered approach using microRNAs to address how the olfactory system specifies the unique 0R identity of a large number of sensory neurons.

MicroRNAs are short, single stranded RNAs that regulate gene expression by binding to target mRNAs. Drosophila has over 238 different microRNAs, several of which have been associated with neural development. Using two-tiered approach using miRNAs as the primary screen tool resulted in the identification of miRNAs whose over-expression in the olfactory neurons disrupts the olfactory receptor expression. MicroRNA-target pairing is determined via complementary base pairing through a short seed sequence. Using bioinformatic target prediction of the seed sequence of each positive hit in our primary screen provided a list of candidates for a secondary RNAi-based screen. The predicted miRNA targets were set as priority based on expression pattern, and if multiple miRNAs give the same phenotype, overlapping target predictions can be given higher priority in the follow-up screen. The two-tiered system allows to rapidly close in on individual target genes involved in a phenomenon of interest in sometimes < 200 crosses instead of the thousands required for a typical full-coverage RNAi screen. By identification and characterisation of the miRNA targets that contribute to the olfactory receptor expression using two-tiered approach we found previously unknown transcription factor Atf3 playing important role in olfactory receptor expression by screening <50 RNAi lines.