## Introduction

The development of libraries of transgenic RNAi fly stocks (e.g., the [Vienna *Drosophila* RNAi library](http://stockcenter.vdrc.at/control/rnailibrary) and the [Transgenic RNAi Project (TRiP)](http://www.flyrnai.org/TRiP-HOME.html)) has been a tremendous boon to the *Drosophila* community because they permit tissue-specific knockdown of almost all genes in the genome. These resources make it possible to perform genome-wide screens for genes associated with almost any phenotype of interest. Unfortunately, the sheer size of these libraries--often upwards of 20,000 stocks--means that performing such screens remains labor-intensive and tedious. In this paper, we describe our development of a two-tiered screening system comprising an initial pooling screen using miRNA over-expression that generates a list of candidate genes involved in a phenotype of interest and a secondary screen using gene-specific RNAi that narrows this list of candidates to a single responsible target gene.

MicroRNAs are short (i.e., ~22 nucleotide), endogenous, single-stranded RNA molecules that act in the context of a large protein complex to either inhibit translation or induce the degradation of target mRNAs. Since the miRNA-target mRNA relationship is determined largely by a short ~9 nucleotide seed sequence that may occur in many copies scattered over the genome, many miRNAs are capable of down-regulating multiple targets. The specificity of the relationship between a miRNA seed sequence and its complements in the open reading frames and 3'-untranslated regions (3'-UTRs) of target mRNAs is sequence-specific. Several research groups have developed bioinformatic algorithms that take advantage of this aspect of miRNA biology to convert mature miRNA sequences into lists of potential mRNA targets. These lists of candidate targets, however, are plagued by large numbers of false positives partly because the algorithms that generate them cannot fully account for the precise spatial and temporal patterns of miRNA and target mRNA expression.

In other words, a miRNA may be capable of down-regulating a particular target and never get the chance because the two are never simultaneously expressed in the same tissue. It follows conversely that miRNA overexpression in arbitrary tissues using the popular GAL4/UAS system would likely lead to artifactual miRNA-target mRNA pairings. Rather than seeing such artifacts as a potential drawback of using a library of UAS-miRNA stocks, we propose that such pairings can be quite useful.

We decided to carry out our first proof-of-concept two-tiered miRNA overexpression screen in the fly olfactory system. Recently Jafari et al. reported the results of a classical large-scale RNAi screen that identified several transcription factors, which form a combinatorial code that determines the fly's highly stereotyped odorant receptor expression patterns\cite{Brochtrup_Hummel_Alenius_2012}.