### Results :

## Figure 1: Two-tiered miRNA screening approach.
A.Screen flowchart
B. Construction of miRNA library
C. Confirmation of miRNA expression

Two-tiered approach makes use of miRNAs as the primary screen tool. MicroRNA-target pairing is determined via complementary base pairing through a short seed sequence. Using bioinformatic target prediction of the seed sequence of each positive hit in our primary screen provides a list of candidates for a secondary RNAi-based screen. The predicted miRNA targets were set as priority based on expression
pattern, and if multiple miRNAs give the same phenotype, overlapping target predictions are given higher priority in the follow-up screen. The two-tiered system allows to rapidly close in on individual target genes involved in a phenomenon of interest. Schematic representation  of two-tiered miRNA based screen. (Fig 1A).
According to version 17 of miRBase, there are 238 annotated miRNAs in the fly genome (Refer). UAS-miRNA constructs that were previously deposited (refer) as clones with the Drosophila Genomics Resource Center were originally constructed by inserting 400-500 base pairs of DNA surrounding the mature miRNA sequence into the 3â€™-UTR of DsRed and cloning it into the fly transformation vector pUAST. Essentially, this amounts to the cloning of the equivalent of the pri-miRNA and adding it to the transcript of a fluorescent marker protein.  40 of these UAS-miRNA constructs were modified by adding an attB site into each vector to permit site-specific, unidirectional, and highly efficient recombination of the vector into an attP landing site via the action of the Phi-C integrase (refer). This technique helps to reduce the genomic position effects that often plague over-expression experiments with GAL4 and UAS-lines that are randomly inserted into the genome by P-element mediated transgenesis. 
For the rest of the fly miRNAs, modified version of another attB-containing vector called SST13 was used. To prepare this vector for use, we added several restriction sites, inserted DsRed downstream of a 5X UAS sequence, and renamed it pSS-DsRed. For a better fluorescence signal, in some UAS-miRNAs  constructs DsRed was replaced with mCherry. (Fig 1B) For the construction of rest of the library PCR
amplification of pri-miRNAs consisting of 400-500 base pairs surrounding each mature miRNA sequence was performed, TA-cloned into the pGEM-T-Easy vector, and subcloned them into one of the pSS constructs. The clone was confirmed by restriction digestion and sequencing. The contructs were microinjected to produce transgenic flies. To confirm miRNA over-expression, the UAS-miRNA fly lines were crossed to GMR-GAL4, which drives expression in the eye. DsRed or mCherry fluorescence was taken as a proxy-indicator of miRNA expression. MicroRNA over-expression was also confirmed by qPCR.(data not shown) (fig 1c)

## Figure 2: Primary screen for regulators of OR expression:
A.Genetic scheme for the screen
B. Brain staining for various miRNA O/E showing loss of Or47b, loss of Or92a, and loss of both
C. Mixed probe in situs confirming the Or47b loss with a couple of miR 33 miR-2a-2/2c miR 986 

To determine if miRNAs play a role in OR expression,  UAS-miRNA library of Drosophila stocks was used for tissue-specific over-expression of specific miRNAs.  The standard GAL4-UAS system was used. (refer) To adapt this system in over-expression experiments and to find a way to visualize a limited but specific population of ORNs, the fusions of OR promoters to GFP were used.  The two OR promoter fusion of Or47b and Or92a neurons were selected because the glomeruli they innervate are large and easy to identify. Or92a neurons project to the VA2 glomerulus whereas Or47b neurons project to the VA1lm glomerulus at the ventrolateral edge of the antennal lobe. (refer) The pebbled-GAL4 driver line, that is expressed in peripheral sensory neurons throughout pupation, with the OR promoter fusion to Or47b and Or92a (refer) was used for the tissue-specific over-expression of miRNAs. Fly brain was immunostained for visualizing any loss of GFP expression.

In two separate rounds the effect of over-expression of miRNA was analysed. The individual miRNAs gave 3 distinct class of tissue specific loss of OR phenotype. (Fig B). We found the list of miRNAs that cause exclusively loss of Or47b, loss of Or92a and loss of both the ORs (Fig 2B and Supplementary).  Of 133 UAS-miRNA lines screened, 20 of the miRNAs were found to be lethal. Some over-expression of miRNAs that caused egg laying deficiency, larve or pupae lethalilty or only few (3-4) sick flies eclosed out were left out of screening.(Supplementary). 
In order to rule out the false positives, the experiments were reproduced thrice and mixed probe in situ hybridization for Or47b and Or92a were performed. (See materials and methods). The clear loss Or47b transcript by in situ hybridization validates the immunostaining results produced by over-expression of certain miRNAs. (Fig2 c)

## Figure 3: Secondary RNAi screen for regulators of OR expression:
A. Identification of Atf3 as a gene of interest (miRNA target lists with overlap analysis)
B. Brain staining of Pbl-Gal4>Atf3-IR, OR promoter fusions
C. Mixed probe in situs for Or47b and Or92a on Pbl-Gal4>Atf3-IR and controls
D. miRNA overexpression reduces atf3 expression (Pbl-Gal4>UAS-miR-X; GFP::atf3 and controls)

In both mammals and insects, ORNs majorly express a single OR gene. To identify the targets that are necessary for the proper 0R expression in Drosophila, the list of miRNAs that gave the selective loss of OR expression were analyzed. The possible RNAi hits were based on the
following criterion, first- the over-lapping targets in both UTR and ORF among the miRNAs that showed the same phenotype were set as the highest priority, second-the over-lapping targets among different miRNAs showing the highest complementary seed sequence pairing was used and  the number of hydrogen bond pairing were counted for the strength among individual targets. The bioinformatics programs
Targetscan fly, miRanda and Microcosm were used to narrow on > 40 RNAi lines for the follow up screen. (Fig 3a, Supplementary fig 2)

The RNAi were expressed in post mitotic ORNs by pebbled GAL4 and OR expression was visualized by direct OR promoter fusions to GFP as described previously. In two separate rounds, the RNAi effect
on the OR expression of two OR classes namely Or47b and Or92a was used, which led to identification of previously unknown role of new transcription factor, Atf3 which resulted in the strong and highly penetrant loss of Or47b expression. (Fig 3b). 

In order to exclude out false positives caused by off target effect, multiple atf3 RNAi lines were analyzed from BDSC and VDRC.  All the constructs gave rise to identical OR loss phenotype, supporting the specific knocks down of Atf3 transcription factor. Mixed Or47b probe
and Or92a probe in situ hybridization was performed on atf3 knock down flies to validate the selective Or47b loss. (Fig 3c)

For the establishing a direct role between atf3 and over-expression of certain miRNA  that resulted in  selective knock down of Atf3 transcript in Or47b neurons, we combined Pebbled GAL4 driving UAS-miRNA-2491 with atf3::EGFP construct. We hypothesized that the miRNA over- expression should also result in the loss of Atf3 expression. Immunostaining was performed and result--- (yet to complete). (Fig 3d)

## Figure 4: Atf3 is required in the antennae for Atf3 expression:
A. Antennal expression of atf3 (in situ)
B. Atf3 is expressed in Or47b neurons (Or47b-Gal4>TdTomato; GFP::Atf3)
C. Rescue of Atf3 in Orco neurons rescues Or47b expression (in situ and/or IHC)

The positive hit, Atf3 that adds to the list of transcription factors that play vital role in OR expression ( refer alenius, ray et al, 2007, song et al 2012)  is a single fly ortholog of the vertebrate activating transcription
factor 3. Atf3 belongs to the family of basic leucine zipper (bZIP) transcription factors. Atf3 is known to regulate the metabolic and immune system homeostasis in fly (refer uhlirova). The construct that expresses atf3::EGFP fusion protein under the natural atf3 regulatory sequence has shown moderate Atf3::EGFP expression in the brain, ring gland, eye/antennal imaginal disc and wing disc of third instar larva of atf3 BAC clone.(refer uhlirova 2012). The diversified tissues expressesing Atf3, hints the possibility of its regulatory role in them.

We next asked whether the TF expression co-related with the OR expression in the adult antennae. The cDNA was prepared using Canton S antennal RNA devoid of head and body. Using antennal origin cDNA as a
template, ~800 bp of cDNA was PCR amplified by Atf3 specific primers. The amplified product was TA cloned to pGEMT-easy vector and confirmed by sequencing prior preparing riboprobes. The in situ hybridization on the wild type antennae with minor changes in washing temperature, and hybridization time (see material and methods for details) was performed. Moderately low level of Atf3 expression was seen in the adult fly antennae. (Fig.4a)

As the knock down of Atf3 resulted in the loss of Or47b expression, we hypothesized that in order to cause the selective loss, Atf3 transcription factor must ideally be present in Or47b neurons. We used Or47b-GAL4, and combined it with UAS-Td-Tomato construct, so that Td-tomato signal is a result of Or47b-GAL4. Atf3::EGFP clone was used to confirm the hypothesis. The co-expression of GFP with Td-tomato Or47b expressing neurons by antennal staining suggest that (yet to complete experiment ) (Fig 4B)

Rescue - Yet to complete the experiment. (4c)