# Summary

Large scale genetic screening is tedious and time-consuming. To address this problem, we propose a novel two-tiered screening system comprising an initial "pooling" screen that identifies miRNAs whose tissue-specific over-expression causes a phenotype of interest followed by a more focused secondary screen that uses gene-specific RNAi. As miRNAs inhibit translation or direct the destruction of their target mRNAs, any phenotype observed with miRNA over-expression can be attributed to the loss-of-function of one or more target mRNAs. Since miRNA-target pairing relies on a ~9 nucleotide seed sequence, bioinformatic target prediction for the miRNAs identified in the initial screen provides a list of candidates for the secondary RNAi-based screen. These predicted miRNA targets can be prioritized based on expression pattern, and if multiple miRNAs give the same phenotype, overlapping target predictions can be given higher priority in the follow-up screen. This two-tiered system allows us to rapidly identify individual target genes involved in a phenomenon of interest, often in less than 200 crosses.

Since miRNA seed sequences are short, miRNA misexpression will likely uncover artifactual miRNA-target relationships. Rather than focus on the biology of miRNAs themselves, we are using miRNAs as a tool to accelerate genetic screening. Here we demonstrate the effectiveness of this method by identifying miRNAs that alter *Drosophila* odorant receptor expression. With subsequent miRNA target prediction and follow-up RNAi screening we identify and validate a novel role for the transcription factor Atf3 in the expression of the socially relevant receptor Or47b.