## Summary

Large scale genome-wide genetic screening is tedious and time-consuming, especially for phenotypes that demand complicated analyses. In an effort to address this problem, we propose a novel two-tiered screening system comprising an initial “pooling” screen that identifies a miRNA or miRNAs whose tissue-specific over-expression causes a phenotype of interest and a more focused secondary screen using gene-specific RNAi. As miRNAs inhibit translation or direct the destruction of their target mRNAs, any phenotype observed with miRNA over-expression can be attributed to the loss-of-function of one or more target mRNAs. Since miRNA-target pairing is determined via complementary base pairing through a ~9 nucleotide "seed" sequence, bioinformatic target prediction for each positive hit in the initial miRNA overexpression screen provides a list of candidates for the secondary RNAi-based screen. These predicted miRNA targets can be prioritized based on expression pattern, and if multiple miRNAs give the same phenotype, overlapping target predictions can be given higher priority in the follow-up screen. This two-tiered system allows us to rapidly identify individual target genes involved in a phenomenon of interest in sometimes less than 200 crosses instead of the thousands required for a typical full-coverage RNAi screen.

MicroRNA seed sequences are short enough to ensure that miRNA misexpression often uncovers artifactual miRNA-target relationships. Thus, rather than focus on the biology of miRNAs themselves, we are using miRNAs as a tool to accelerate genetic screening. Here we demonstrate the effectiveness of this method by identifying miRNAs that alter *Drosophila* odorant receptor expression. With subsequent miRNA target prediction, follow-up RNAi screening, and mutant analysis, we identify and validate a novel role for the transcription factor Atf3 in the expression of the socially relevant receptor Or47b.